<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawForm</class>
 <widget class="QDialog" name="DrawForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Draw Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/draw.png</normaloff>icons/draw.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="structure">
      <attribute name="title">
       <string>Structure</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="listViewNodeStructure"/>
         </item>
         <item>
          <widget class="QListWidget" name="listViewElementStructure"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Node</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QRadioButton" name="radioButtonFirstClass">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>1º Class</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonSecondClass">
                <property name="text">
                 <string>2º class</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonTirdClass">
                <property name="text">
                 <string>3º class</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonSemiRigidClass">
                <property name="text">
                 <string>Semi Rigid</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLineEdit" name="lineEditNodeX">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>X value</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditNodeY">
                <property name="placeholderText">
                 <string>Y value</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditNodeRotation">
                <property name="placeholderText">
                 <string>Rotation (º)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonNodeDraw">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="text">
                 <string>Draw</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Element</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="4" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="text">
                 <string>Draw</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="lineEditSecondNodeY">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>2nd Node Y value</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineEditFistNodeX">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>1st Node X value</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEditFistNodeY">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>1st Node Y value</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEditSecondNodeX">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>2nd Node X value</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadings">
      <attribute name="title">
       <string>Loadings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0" rowspan="2">
        <widget class="QListWidget" name="listWidgetNodeLoadings"/>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxLoadingType">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Loading Type</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Nodal Forces</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_7">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Uniform</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_8">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Linear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBoxGenerateNode">
         <property name="title">
          <string>Generate Node</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Loadings</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLineEdit" name="lineEditForceX">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Loading in X (Fx)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="lineEditForceY">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Loading in Y (Fy)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLineEdit" name="lineEditMoment">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Moment (M)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Position</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLineEdit" name="lineEditNodeX_2">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>X value</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditNodeY_2">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Y value</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLineEdit" name="lineEditNodeRotation_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Rotation (º)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonNodeDraw_2">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Draw</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditNodeX</tabstop>
  <tabstop>lineEditNodeY</tabstop>
  <tabstop>lineEditNodeRotation</tabstop>
  <tabstop>lineEditFistNodeX</tabstop>
  <tabstop>lineEditFistNodeY</tabstop>
  <tabstop>lineEditSecondNodeX</tabstop>
  <tabstop>lineEditSecondNodeY</tabstop>
  <tabstop>pushButtonNodeDraw</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>radioButtonFirstClass</tabstop>
  <tabstop>radioButtonSecondClass</tabstop>
  <tabstop>radioButtonTirdClass</tabstop>
  <tabstop>radioButtonSemiRigidClass</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
